testgrid/cmd/configurator/prow.go

```diff
@@ -40,6 +40,7 @@ const testgridEmailAnnotation = "testgrid-alert-email"
 const testgridNumColumnsRecentAnnotation = "testgrid-num-columns-recent"
 const testgridAlertStaleResultsHoursAnnotation = "testgrid-alert-stale-results-hours"
 const testgridNumFailuresToAlertAnnotation = "testgrid-num-failures-to-alert"
+const testgridDaysOfResultAnnotation = "testgrid-days-of-result"
```

Comment:
nit: Should be 'results' and not 'result', here and elsewhere

Reply:
Renamed it.